pytket/pytket/backends/backendinfo.py

```diff
@@ -13,13 +13,127 @@
 # limitations under the License.
 
 """ BackendInfo class: additional information on Backends """
+
+from ast import literal_eval
 from dataclasses import dataclass, field, asdict
 from typing import Any, Dict, List, Optional, Set, cast, Tuple, Union
 
 from pytket.routing import Architecture, FullyConnected  # type: ignore
 from pytket.circuit import Node, OpType  # type: ignore
 
 
+def _serialize_all_node_gate_errors(
+    d: Optional[Dict[Node, Dict[OpType, float]]]
+) -> Optional[Dict[str, Dict[int, float]]]:
+    if d is None:
+        return None
+    return {
+        str(n.to_list()): {ot.value: err for ot, err in errs.items()}
```

Comment:
I would suggest either using `tuple(n.to_list())` or using a list of pairs instead of the dictionary (to avoid stringifying a list)

Reply:
That doesn't quite work because `tuple(n.to_list())` is something like `("node", [0])` which is not hashable, but we could do something similar by going one level down and converting the inner list to a tuple.

Reply:
I think the list of pairs is better then, its what we do with other 1-1 maps